build.gradle

```diff
@@ -15,6 +16,7 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        google()
```

Comment:
Are we using this repository?

Reply:
it is required by Latest Compat Libraries